src/vcCamera.cpp

```diff
@@ -696,21 +697,20 @@ void vcCamera_HandleSceneInput(vcState *pProgramState, udDouble3 oscMove, udFloa
 
   if ((!ImGui::GetIO().WantCaptureKeyboard || isFocused) && !pProgramState->modalOpen)
   {
-    keyboardInput.y += io.KeysDown[SDL_SCANCODE_W] - io.KeysDown[SDL_SCANCODE_S];
-    keyboardInput.x += io.KeysDown[SDL_SCANCODE_D] - io.KeysDown[SDL_SCANCODE_A];
-    keyboardInput.z += io.KeysDown[SDL_SCANCODE_R] - io.KeysDown[SDL_SCANCODE_F];
+    keyboardInput.y += io.KeysDown[vcKey::Get("Forward")] - io.KeysDown[vcKey::Get("Backward")];
+    keyboardInput.x += io.KeysDown[vcKey::Get("Right")] - io.KeysDown[vcKey::Get("Left")];
+    keyboardInput.z += io.KeysDown[vcKey::Get("Up")] - io.KeysDown[vcKey::Get("Down")];
```

Comment:
I feel less happy about that because in a lot of cases it unravels to for each check also doing ImGui::GetIO() and anyway doesn't need any of the functionality of pressed()